docs/hub/tutorial-add-library.md

```diff
@@ -0,0 +1,184 @@
+# Integrate your library with the Hub
+
+The Hugging Face Hub aims to facilitate the sharing of machine learning models, checkpoints, and artifacts. This endeavor includes integrating the Hub into many of the amazing third-party libraries in the community. Some of the incredible libraries already integrated include [spaCy](https://spacy.io/usage/projects#huggingface_hub), [AllenNLP](https://allennlp.org/), and [timm](https://rwightman.github.io/pytorch-image-models/), among many others. Integration means users can download and upload files to the Hub directly from your library. We hope you will integrate your library and join us in democratizing artificial intelligence for everyone!
+
+Integrating the Hub with your library provides many benefits, including:
+
+- Free model hosting for you and your users.
+- Built-in file versioning - even for huge files - made possible by [Git-LFS](https://git-lfs.github.com/).
+- All public models are powered by our lightning-fast [Inference API](https://api-inference.huggingface.co/docs/python/html/index.html).
+- In-browser widgets allow users to interact with your hosted models directly.
+
+This tutorial will help you integrate the Hub into your library, so your users can benefit from all the features offered by the Hub.
+
+Before you begin, we recommend you create a [Hugging Face account](https://huggingface.co/join) from which you can manage your repositories and files. 
+
+## Installation
+
+1. Install the `huggingface_hub` library with pip in your environment:
+
+   ```bash
+   python -m pip install huggingface_hub
+   ```
+
+2. Once you have successfully installed the `huggingface_hub` library, log in to your Hugging Face account:
+
+   ```bash
+   huggingface-cli login
```

Comment:
See my comment above about logging in via notebooks (maybe not a common thing for this use case though, so we might just want to add a note)